dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -777,7 +780,10 @@ void DAGStorageInterpreter::buildLocalStreams(DAGPipeline & pipeline, size_t max
                 throw;
             }
         }
-        pipeline.streams.insert(pipeline.streams.end(), current_pipeline.streams.begin(), current_pipeline.streams.end());
+    }
+    for (int i = 0; i < static_cast<int>(max_streams); i++)
```

Comment:
I think we don't need to use `MultiplexInputStream` for non-partition table scan or partition table scan with only one partition.

Reply:
I agree with you, the only trouble is the code will be more dirty when two cases are not in a universal code path.